src/encoded/tests/test_schema_document.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict
+
+import pytest
+from webtest.app import TestApp
+
+from .utils import post_item
+
+
+@pytest.fixture
+def document(
+    testapp: TestApp,
+    test_consortium: Dict[str, Any],
+    test_submission_center: Dict[str, Any],
+) -> Dict[str, Any]:
+    item = {
+        "consortia": [test_consortium["uuid"]],
+        "submission_centers": [test_submission_center["uuid"]],
+        "description": "Some document",
+    }
+    return post_item(testapp, item, "Document")
+
+
+def test_document_post(document: Dict[str, Any]) -> None:
+    """Ensure Document properties POST."""
+    pass
```

Comment:
You might want to assert something basic in these tests ie: description matches or existence of calc prop. Same deal for other tests of this nature.